scripts/mbedtls_framework/c_wrapper_generator.py

```diff
@@ -25,12 +24,34 @@ def c_declare(prefix: str, name: str, suffix: str) -> str:
     return prefix + name + suffix
 
 
-WrapperInfo = typing.NamedTuple('WrapperInfo', [
+WrapperInfo = NamedTuple('WrapperInfo', [
     ('argument_names', List[str]),
     ('guard', Optional[str]),
     ('wrapper_name', str),
 ])
 
+def strip_indentation(in_str: str, new_lines: int = 1, indent_lv: int = 0) -> str:
+    """Return a whitespace stripped str, with configurable whitespace in output.
+
+    The method will remove space-character indentation from input string.
+    It will also remove all new-lines around the text-block.
+    The output indentation can be configured by indent_lv, and will use blocks
+    of 4 spaces.
+    At the end of the string a `new_lines` amount of empty lines will be added.
+    """
+
+    _ret_string = in_str.strip('\n')
+
+    # Count empty spaces in beggining of each line. The smallest non-zero entry
+    # will be used to clean up input indentation.
+    _common_indents = min([len(n) for n in re.findall(r'(?m)^ +', in_str)])
+    if _common_indents:
+        _ret_string = re.sub(r'(?m)^ {{{indent}}}'.format(indent=_common_indents),
+                             '', _ret_string)
```

Comment:
This errors out if no line is indented. E.g.
```suggestion
    indents = [len(n) for n in re.findall(r'(?m)^ +', in_str)]
    if indents:
        _ret_string = re.sub(r'(?m)^ {{{indent}}}'.format(indent=min(indents)),
                             '', _ret_string)
```